### DISCUSSION TITLE
Icon in table header

### DISCUSSION THREAD

**Discussion by micraux** [filamentphp/filament#8438] [Help] [answered]
I need to have very small columns in a table, because the user will use a smartphone and I have 5 columns containing only 2-digit values.

So I came up with the idea of using an icon as the column header, and modified the `->label()` like this:
```
->label(
    new HtmlString('
        <span class="h6 w-6 block stroke-1.5 stroke-current">
            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
                <path strokeLinecap="round" strokeLinejoin="round" d="..." />
            </svg>
        </span>
    ')
)
```
It does work, but I am wondering: is there a more elegant solution, by calling directly the heroicon name for example?

**Comment by leandrocfe** [comment, marked as answer]
```php
->label(new HtmlString(Blade::render('<x-heroicon-o-cube class="w-6 h-6" />')))
```
https://filamentphp.com/docs/3.x/support/icons#overview

**Reply by micraux** [reply]
Perfect, thank you!

**Reply by leandrocfe** [reply]
I believe that if you keep topics open with the "answered" tag, it can help other users

**Reply by micraux** [reply]
Ok, fine, I reopen it

**Reply by dissto** [reply]
> Ok, fine, I reopen it

Don't forget to mark the answer 😁

**Reply by micraux** [reply]
Oh yes, I forgot. Thanks!

**Reply by Congminh0707** [reply]
Great!

**Reply by hedihadi** [reply]
> ```
> ->label(new HtmlString(Blade::render('<x-heroicon-o-cube class="w-6 h-6" />')))
> ```
> 
> https://filamentphp.com/docs/3.x/support/icons#overview

i don't know if i'm a silly goose or maybe this'll help someone else.
if you have `->translateLabel()`, this will give `illegal offset type` error, you have to remove it.
if you've set `translateLabel` to true globally (inside your service provider) then you need to add `->translateLabel(false)` to the column.
